test/metabase/query_processor/middleware/cache_test.clj

```diff
@@ -323,7 +327,32 @@
                        (-> cached-result
                            (dissoc :cached :updated_at)
                            (m/dissoc-in [:data :results_metadata :checksum])))
-                    "Cached result should be in the same format as the uncached result, except for added keys")))))))))
+                    "Cached result should be in the same format as the uncached result, except for added keys"))))))))
+  (testing "Cached results don't impact average execution time"
```

Comment:
nit: space

Reply:
space where? I'm not following.

Reply:
actually, damn. there's a space between each `testing` clause in all the api tests and this is very inconsistent within QP tests. I guess we should just follow the format within each file or whack it entirely per file, not piecemeal like this, so never mind on this one

Reply:
can you link to an example? I'm not following where a space would logically go?

Reply:
https://github.com/metabase/metabase/blob/9587fdbc4e98b06511575fc5dfb60328fbb7957e/test/metabase/api/collection_test.clj#L73

Reply:
ah newlines. i see. asking around if we have a standard. I don't care one way or the other.

Reply:
memorializing side channel dealio: "season to taste"